src/picknik_ur_mujoco_config/config/config.yaml

```diff
@@ -42,7 +42,6 @@ objectives:
       - "moveit_studio::behaviors::MTCCoreBehaviorsLoader"
       - "moveit_studio::behaviors::ServoBehaviorsLoader"
       - "moveit_studio::behaviors::VisionBehaviorsLoader"
-      - "moveit_studio::behaviors::ConverterBehaviorsLoader"
```

Comment:
In `main` we should be able to keep `ConverterBehaviorsLoader`, which is needed for the latest VFC examples